test/run_pass/test_buffer.js

```diff
@@ -141,3 +141,4 @@ assert.equal(buff16.readInt8(2), 13);
 assert.equal(buff16.readInt8(3), 13);
 
 assert.equal(Buffer(new Array()).toString(),'');
+assert.equal(new Buffer(1).readUInt16LE({}, true),0);
```

Comment:
I added a test case for readUInt8, which is called inside readUInt16LE with offset, -1. Thank you.